bin/www

```diff
@@ -1,7 +1,10 @@
 const app = require('../app');
 const debug = require('debug')('surfboard:server');
+const models = require('../models');
```

Comment:
I'm thinking we should just have an npm start target that simply runs server/index.js for startup